spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -46,7 +46,7 @@ describe('document-capture/components/document-capture', () => {
     userEvent.click(submitButton);
 
     const confirmation = await findByText(
-      'Finished sending: {"front_image":"data:image/png;base64,","back_image":"data:image/png;base64,","selfie":"data:image/png;base64,"}',
+      'Finished sending: {"front":"data:image/png;base64,","back":"data:image/png;base64,","selfie":"data:image/png;base64,"}',
```

Comment:
Maybe it will happen after #4082, but these should no longer be data image URLs, right? Just file attachments?

Reply:
> Maybe it will happen after #4082, but these should no longer be data image URLs, right? Just file attachments?

After #4082, yeah. Now that it's merged, I can rebase and this will be more like what you were expecting.

Reply:
> > Maybe it will happen after #4082, but these should no longer be data image URLs, right? Just file attachments?
> 
> After #4082, yeah. Now that it's merged, I can rebase and this will be more like what you were expecting.

Now reflected in rebased 88b812082.